joomla/joomla-cms
# Allow skins in TinyMCE

This will enable TinyMCE to use skins
# What problem is being solved

Right now in Joomla! is hard to customize the look of the tinyMCE editor (needs some core hacks).
This PR will enable the use of skins just by copying them in the respective folder.
# B/C

There shouldn’t be any backward compatibility breaks here.
In the options of tinyMCE instead of having the default as the only option in drop down list, all the available skins will be listed by the respected names. Also the string that translates to default is useless but I didn’t deleted as I don’t know how language files are treated…
Line 102@ en-GB.plg_editors_tinymce.ini: PLG_TINY_FIELD_VALUE_DEFAULT="Default"
# How to test

Need to set editor to tinymce:
1. Apply the patch with patchtester.
2. Edit an article. Everything should be ok.
3. Go to http://skin.tinymce.com and select the preset charcoal or tundra and download it. (lightgray is the one that ships with Joomla!)
![screenshot 2014-09-07 15 10 54](https://cloud.githubusercontent.com/assets/3889375/4178260/a6eda45e-3688-11e4-9a59-17115eda7569.png)
4. Extract the skin in /media/editors/tinymce/skins
5. Go to plugins->tinyMCE and select the skin you just copied.
![screenshot 2014-09-07 15 12 34](https://cloud.githubusercontent.com/assets/3889375/4178262/b0f659e6-3688-11e4-93a3-8382cb0b5922.png)
6. Edit an article with your new skin!
![screenshot 2014-09-07 15 13 21](https://cloud.githubusercontent.com/assets/3889375/4178263/b728b368-3688-11e4-9df6-5e976bedb9bc.png)
